package.json

```diff
@@ -11,6 +11,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.19.2"
+    "dotenv": "^16.4.5",
+    "express": "^4.19.2",
+    "mongodb": "^6.8.0",
+    "mongoose": "^8.6.0"
```

Comment:
dont need mongodb and mongoose. i think its better to work with mongodb. - look on the file that Haim send in the Model. 
and after you change to mongodb, need to change the models